redaxo/src/core/lib/util/i18n.php

```diff
@@ -449,4 +451,22 @@ private static function loadAll($locale)
 
         self::$loaded[$locale] = true;
     }
+
+    /**
+     * @param string $locale Locale
+     *
+     * @return string the validated locale
+     *
+     * @psalm-taint-escape file
+     */
+    private static function validateLocale($locale)
+    {
+        if (5 != strlen($locale)) {
+            throw new rex_exception("Invalid locale $locale");
+        }
+        if (!preg_match('/^[a-z]{2}_[a-z]{2}$/', $locale)) {
+            throw new rex_exception("Invalid locale $locale");
```

Comment:
```suggestion
            throw new rex_exception('Invalid locale "'.$locale.'"');
```

So ist es insbesondere bei leerem String besser verständlich.